fiftyone/factory/repos/execution_store.py

```diff
@@ -104,15 +126,51 @@ def delete_store(self, store_name: str) -> int:
 
     @abstractmethod
     def set_key(
-        self, store_name: str, key: str, value: Any, ttl: Optional[int] = None
+        self,
+        store_name: str,
+        key: str,
+        value: Any,
+        ttl: Optional[int] = None,
+        policy: str = "persistent",
     ) -> KeyDocument:
-        """Set a key in a store.
+        """
```

Comment:
This doc string should move / merge with the service set_key. Since repos are all internal no one should really ever need to read this.